docs/rancher/csi-driver.md

```diff
@@ -277,6 +277,121 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
 
 1. You can now create a PVC based on this new **StorageClass**, which utilizes the **Host StorageClass** to provision volumes on the bare-metal Harvester cluster.
 
+## RWX volumes support
+
+### Prerequisites
+
+:::note
+
+To use this feature, you need to ensure the host harvester cluster version is **v1.4.0** or later.
+
+:::
+
+Before creating RWX volumes on the guest cluster, you need to create a corresponding RWX StorageClass on the host cluster.
+
+1. Create a RWX StorageClass on the host
+  ```
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: longhorn-rwx
+  provisioner: driver.longhorn.io
+  allowVolumeExpansion: true
+  reclaimPolicy: Delete
+  volumeBindingMode: Immediate
+  parameters:
+    numberOfReplicas: "3"
+    staleReplicaTimeout: "2880"
+    fromBackup: ""
+    fsType: "ext4"
+    nfsOptions: "vers=4.2,noresvport,softerr,timeo=600,retrans=5"
+  ```
+
+  ![](/img/v1.4/rancher/create-rwx-sc-host-cluster-01.png)
+
+  ![](/img/v1.4/rancher/create-rwx-sc-host-cluster-02.png)
+
+  ![](/img/v1.4/rancher/create-rwx-sc-host-cluster-03.png)
+
+2. Ensure the RBAC settings are up-to-date
+
+  To support RWX feature, the harvester CSI driver needs to have the new RBAC settings. You can follow the steps below to check the RBAC settings.
```

Comment:
```suggestion
- The RBAC settings are up-to-date.

  The Harvester CSI driver requires must have the new RBAC settings to support RWX volumes. To check the RBAC settings, run the command `kubectl get clusterrole harvesterhci.io:csi-driver -o yaml`.
```

Reply:
Which RBAC settings are you referring to? The Harvester doc does not contain any topics about RBAC settings. Please provide links to related information.

Reply:
The RBAC is the `Role-Based Access Control`. That is permission control to limit the resource operations (like `GET`, `LIST`, `CREATE` , `DELETE` and `UPDATE`).
The CSI driver needs some new permission to check/update the specific resource.

I will add some explanation here and provide the link.

Thanks!

Reply:
Updated. Please help to check, thanks!